src/templates/governance-page/DaoGovernanceDetails/DaoDelegation.tsx

```diff
@@ -1,17 +1,54 @@
-import { DAO } from "./types"
 import { FC } from "react"
+import { FixedNumber } from "ethers"
+import { gql } from "graphql-request"
 import { HStack, Stack } from "@chakra-ui/react"
 import { PageSection, ResponsiveStack } from "../../../components/PageSection"
 import { BodyLg, BodyMd, BodySm, H3, H4, LabelMd } from "../../../components"
 import ExternalButtonLink from "../../../components/Buttons/ExternalButtonLink"
 import { ExternalLinkHref } from "../../../components/Navbar/types"
 import Card from "../../../components/Card"
 import Progress from "../../../components/Progress"
+import useQuery from "../../../hooks/useQuery"
+import { formatTokenAmount } from "../../../utils"
+import { T_NETWORK_SUBGRAPH_URL } from "../../../config/subgraph"
+import { DAO } from "./types"
+
+const T_TOTAL_SUPPLY_IN_WEI = "10000000000000000000000000000"
 
 const DaoDelegation: FC<{ selectedDao: DAO }> = ({ selectedDao }) => {
-  const totalDelegated = "2,000,000 T"
-  const maxDelegation = "100,000,000"
-  const percentDelegated = 20
+  const { isFetching, data, error } = useQuery<{
+    daometric: { liquidTotal: string; stakedTotal: string }
+    epoches: { totalStaked: string }[]
```

Comment:
can we rename to `epochs`? 😅

Reply:
We can add an alias like: `epochs: epoches {...}`- the Graph creates the plural form for entities.